### DISCUSSION TITLE
Jose 3.x version of the Jose 2.x verify feature

### DISCUSSION THREAD

**Discussion by joelpowers** [panva/jose#195] [Q&A]
Hey - I was using jose 2.x and the verify feature - was working great.  I want to upgrade to 3.x but am having trouble - seems there isn't the equivalent thing in 3.x.
 
// working - 2.x typescript
import * as jose from 'jose';
...
    this.verifyKey(jwt, key);
Where jwt is a string.  key is an object something like:
{
      alg: 'ES256',
      created_at: 1560466143,
      crv: 'P-256',
      expired_at: null,
      kid: 'snip snip snip',
      kty: 'EC',
      use: 'sig',
      x: 'snip snip snip',
      y: 'snip snip snip-snip snip snip',
    }
 
What is the equivalent in 3.x? I've looked at jwtVerify but the 2nd paramber seems to be something else and there are verifyOptions, not sure what to use there.
Thanks in advance!!

**Comment by JBaczuk** [comment]
Have you tried parsing the object using `parseJwk`?

`const { parseJwk } = require('jose/jwk/parse')`

**Reply by joelpowers** [reply]
I downgraded to 2.x but will try ⬆️ tonight - thanks!
